geo-types/src/lib.rs

```diff
@@ -60,12 +61,17 @@
 //!
 //! The following optional [Cargo features] are available:
 //!
+//! - `std`: Enables use of the full `std` library. Enabled by default.
 //! - `approx`: Allows geometry types to be checked for approximate equality with [approx]
 //! - `arbitrary`: Allows geometry types to be created from unstructured input with [arbitrary]
 //! - `serde`: Allows geometry types to be serialized and deserialized with [Serde]
 //! - `use-rstar_0_8`: Allows geometry types to be inserted into [rstar] R*-trees (`rstar v0.8`)
 //! - `use-rstar_0_9`: Allows geometry types to be inserted into [rstar] R*-trees (`rstar v0.9`)
 //!
+//! This library can be used in `#![no_std]` environments if the default `std` feature is disabled. At
+//! the moment, the `arbitrary` and `use-rstar_0_8` features cannot be used without `std`. This may
```

Comment:
```suggestion
//! the moment, the `arbitrary` and `use-rstar_0_8` features require `std`. This may
```

Reply:
Applied in [5f0bd98](https://github.com/georust/geo/pull/936/commits/5f0bd98339452ec5f2e9210ce3e33060f11c9b66)